tests/e2e/tests/pilot/testdata/networking/v1alpha3/sidecar-scope-ns1-ns2.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: Sidecar
+metadata:
+  name: sidecar-scope-ns1-ns2
+spec:
+  ingress:
+  - port:
+      number: 12345
+      protocol: HTTP
+      name: inbound-http
+    defaultEndpoint: ":80"
+  egress:
+  - port:
+      number: 23145
+      protocol: TCP
+      name: outbound-tcp
+    bind: 7.7.7.7
+    hosts:
+    - "*/bookinfo.com"
+    # bookinfo.com is on port 9999.
+  - hosts:
+    - "ns1/*"
+    - "*/*.tcp.com"
```

Comment:
Can you add few more entries - to cover all supported forms ( at least the most common case - ns1/bookinfo.ns1 using a standard service in a namespace ).

Reply:
We dont support the name.namespace. For now, its just namespace/FQDN. In a separate PR I will do the namespace/name but that requires more work in pre processing. name.namespace cannot be supported as there is no way to distinguish a k8s host from a non k8s host that has only two pieces in its dns name.